src/contractInterface/marketplace/v00_adapter.js

```diff
@@ -310,11 +310,18 @@ class V00_MarkeplaceAdapter {
         ipfsHash = e.returnValues.ipfsHash
         createdAt = timestamp
         break
+      case 'OfferAccepted':
```

Comment:
No need to update status in case of OfferAccepted and OfferDisputed.
Since the Offer data is still onchain at this stage (e.g. the Offer data has NOT been deleted from onchain), when rawOffer is loaded at line 291 above, it will return an object with status populated.